src/mlpack/tests/loss_functions_test.cpp

```diff
@@ -871,5 +871,39 @@ BOOST_AUTO_TEST_CASE(SoftMarginLossTest)
   BOOST_REQUIRE_EQUAL(output.n_cols, input.n_cols);
   CheckMatrices(output, expectedOutput, 0.1);
 }
+/**
+ * Simple test for the mean absoulte error function.
+ */
+BOOST_AUTO_TEST_CASE(MeanAbsoluteErrorTest)
+{
+  arma::mat input1, target1, output, input2, target2
+  MeanAbsoluteError<> module(false)
```

Comment:
@iamshnoo i should also change this to be true right ?

Reply:
Well, actually it would be better if you change the default value of the parameter in the constructor as ```const bool reduction=true```, and then just call it here as MeanAbsoluteError<> module. Beacuse that would align with the PR I mentioned above.

Reply:
sure

Reply:
https://github.com/mlpack/mlpack/pull/2603/files#diff-2bf18f53586f3f00c5c4d04a7293280eR43
that is exactly the case right now.

Reply:
also changed in the test file.